src/main/scala/io/iohk/ethereum/cli/CliCommands.scala

```diff
@@ -74,6 +78,23 @@ object CliCommands extends SecureRandomBuilder {
       }
     }
 
+  private val EncryptKey: Command[String] =
+    Command(name = encryptKeyCommand, header = "Encrypt private key") {
+
+      val privateKey = Opts
```

Comment:
Shouldn't common options be defined outside commands? Or maybe even all options should replace current string constants in lines ~17-25?

Reply:
I prefer to leave string constants because it is convenient to use them in tests